pkg/controller/common.go

```diff
@@ -47,6 +47,9 @@ const (
 	maxRetry = 10
 	// DefaultTimeUntilDeletion = time.Duration(5) * time.Minute
 	DefaultTimeUntilDeletion = time.Duration(30) * time.Second
+	defaultRetryDuration     = time.Duration(1) * time.Second
```

Comment:
np: Blank line between global and local symbols definitions. Global and local symbols should also be grouped together (i.e. move `maxRetry` below `DefaultTimeUntilDeletion`).